hedera-node/src/main/java/com/hedera/services/store/models/Account.java

```diff
@@ -258,6 +303,14 @@ public boolean isSmartContract() {
 		return isSmartContract;
 	}
 
+	public void setDetached(boolean val) {
```

Comment:
I think that we can change the name of the parameter to something more precise. "val" is somewhat obscure